js/index.js

```diff
@@ -74,10 +75,29 @@ const imgCta = document.querySelector('#cta-img');
 imgCta.src = "img/header-img.png";
 imgCta.alt = "Image of a code snippet.";
 
-//
-
+//main-content 
+const mainContent = document.querySelectorAll('.text-content');
+/**top of main content  */
+mainContent[0].querySelector('h4').textContent = siteContent['main-content']['features-h4'];
+mainContent[0].querySelector('p').textContent = siteContent['main-content']['features-content'];
```

Comment:
awesome looks like you are accessing the content values from the `siteContent` object.